src/sync/exercises.nim

```diff
@@ -1,8 +1,8 @@
-import std/[algorithm, json, options, os, sequtils, sets, strformat, strutils,
-            tables]
+import std/[json, options, os, sequtils, sets, strformat, strutils, tables]
 import pkg/parsetoml
 import ".."/cli
 import "."/[probspecs, tracks]
+export tracks.`$`
```

Comment:
This will probably be refactored away later.

Reply:
Good. It looks slightly odd :)